docs/ibexa_products/headless.md

```diff
@@ -0,0 +1,193 @@
+
+## Availability
+
+To start using [[= product_name_headless =]] you must purchase a product license.
+For more information, see [[[= product_name_headless =]] license pricing](https://www.ibexa.co/products/pricing?tab=1).
+You can [contact us](https://www.ibexa.co/products/pricing) or [contact one of our partners](https://www.ibexa.co/partners).
+
+## How it works
+
+### Editorial stage
+
+You access with any web browser from any platform to a rich Back Office, the main place to
+
+- define users and their rights (customers, subscribers, editors, etc.),
+- organize content (content types, fields, tree, tags, languages, and more),
+- edit content in a collaborative workplace with versions and workflows.
+
+Then, content is available to end users through REST, GraphQL, or every output you can imagine like websites or apps.
+
+### Technical backstage
+
+When you have a license, you install [[= product_name_headless =]] through Composer
+on an architecture including at least a web server with PHP and a relational database server.
+For performance, several bricks can be added to your stack such as a reverse proxy or a search engine.
+
+[[= product_name_headless =]] is based on Symfony. Any Symfony developer, or even PHP developer, can quickly learn how to extend it with the help of an online documentation.
+
+By using a version control system and environment variables, you can deploy your configuration and extensions on several environments including [[= product_name_cloud =]].
+
+Standard web APIs and [[= product_name_connect =]] help establish interoperability, even if you aren't an advanced developer.
+
+![Ibexa Headless data inputs and outputs](img/headless.png)
+
+APIs summary:
+
+- The REST and GraphQL APIs give access to access the content in standardized ways.
+- The OAuth 2 Client and Server allow to connect to an SSO or be the SSO.
+- The design engine and its theme templates mechanism allows to serve the content in several shapes.
+- The PHP API opens [[= product_name_headless =]] to extendability to fit your needs.
+  For example, content can be computed, edited or served in specific ways such as
+  scheduled/live imports/exports, automated edition tasks, or specific controllers to communicate with other applications.
+
+## Capabilities and benefits
+
+[[= product_name_headless =]] is a tool box with a Back Office.
+It comes without a default front office.
+You don't lose time to develop a theme for a provided front office before discovering it won't fit your needs.
+No distraction.
+[[= product_name_headless =]] helps you focus on the content, create and organize with its straightforward user interface (UI),
+imagine its inputs/outputs, and implement them with its various layers' APIs.
+
+### Core features
+
+The core of [[= product_name =]] Headless edition is already awesome as it offers everything to structure your content repositories and access them.
```

Comment:
As "everything is awesome" anyway, this can be removed.

```suggestion
The core of [[= product_name_headless =]] offers everything to structure your content repositories and access them.
```

I feel like it misses an adjective to keep a bit of marketing tone. Like "hard core".